### DISCUSSION TITLE
Metadata: keywords from XMP sidecars missing

### DISCUSSION THREAD

**Discussion by Zuuushiiihokkiii** [photoprism/photoprism#4100] [Q&A]
Hi, I'm new to Photoprism and have an issue with metadata:

My setup: I use DigiKam to manage my photo library, and use Photoprism to view the library and to share photos. I have configured DigiKam to both write the metadata to the files and create XMP sidecars. The photo library is on a Ceph file system,  which is also mounted (read-only) on my Photoprism VM. Photoprism is installed with docker-compose, with  ofelia scheduled tasks for daily full scans and video transcoding.

This all works perfectly, the only problem I have encountered is that not all metadata written to the XMP sidecars by DigiKam is correctly recognized by Photoprism. Specifically, DigiKam tags are missing from all files, except JPEGs.

For JPEGs, I can see the DigiKam tags under Details > Subject, as expected. All other file types (e.g. HEIC, MP4, MOV) have nothing under Subject and only keywords generated by Photoprism under Keywords, but the XMP sidecar is listed under Files.

I suspect that the problem is that except for JPEGs the DigiKam tags are not present in the files' metadata, presumably because the formats don't support it, and Photoprism uses only the metadata from the files themselves, not the XMP sidecars. Is this a Photoprism limitation, or a configuration error on my end? Thanks

**Comment by graciousgrey** [comment]
Hi :wave: 

PhotoPrism does read metadata from XMP files as well but it might be that the fields you are using are not supported yet. 
Fields related to hierachical tags for example are not yet supported as we first need to impelemnt hierarchical tags in PhotoPrism.

You can find an overview of supported fields here:

https://www.photoprism.app/kb/metadata
https://docs.photoprism.app/user-guide/library/metadata/
